events/tests.py

```diff
@@ -0,0 +1,55 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from .models import Event
+from datetime import datetime
+
+import pytest
+
+
+DATE_TIME_EARLY = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
```

Comment:
maybe it will be more readable like:
```suggestion
DATE_TIME_START = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
```

Reply:
done